src/core/Link/RouterLink/RouterLink.baseStyles.ts

```diff
@@ -0,0 +1,31 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { font } from '../../theme/reset';
+import { allStates } from '../../../utils/css';
+
+export const RouterLinkStyles = (theme: SuomifiTheme) => css`
+  ${font(theme)('bodyText')}
+
+  &.fi-link--router {
+    ${allStates(`color: ${theme.colors.highlightBase};`)}
+    color: ${theme.colors.highlightBase};
+    text-decoration: none;
+
+    &:focus,
+    &:focus-within {
+      text-decoration: none;
+    }
+
+    &:focus {
+      ${theme.focus.boxShadowFocus}
+    }
+
+    &:hover,
+    &:active {
+      text-decoration: underline;
+    }
+    &:visited {
+      color: ${theme.colors.accentTertiaryDark1};
+    }
+  }
+`;
```

Comment:
This is basically a copy-paste from BaseLink styles with just the class name changed. I think it's a good idea to have a separate class for this component so it can be overridden separately from .fi-link if needed.